lib/apiservers/engine/proxy/storage_proxy.go

```diff
@@ -304,11 +314,13 @@ func (s *StorageProxy) AddVolumesToContainer(ctx context.Context, handle string,
 		flags := make(map[string]string)
 		//NOTE: for now we are passing the flags directly through. This is NOT SAFE and only a stop gap.
 		flags[constants.Mode] = fields.Flags
-		joinParams := storage.NewVolumeJoinParamsWithContext(ctx).WithJoinArgs(&models.VolumeJoinConfig{
-			Flags:     flags,
-			Handle:    handle,
-			MountPath: fields.Dest,
-		}).WithName(fields.ID)
+		joinParams := storage.NewVolumeJoinParamsWithContext(ctx).
```

Comment:
`NewVolumeJoinParamsWithContext(op)`

A search of this diff for "WithContext(ctx)" shows 13 matches so I'm not going to comment on all of them.